numpy/core/src/multiarray/multiarraymodule.c

```diff
@@ -4277,6 +4294,11 @@ normalize_axis_index(PyObject *NPY_UNUSED(self), PyObject *args, PyObject *kwds)
     return PyInt_FromLong(axis);
 }
 
+/* declarations from umathmodule.c */
```

Comment:
If following the separate `umathmodule.h` file idea above, these lines could go in there too.

Reply:
cleanup up via removal and refactoring, without a umathmodule.h

Reply:
on second thought umathmodule.h would be better